mlkem/native/riscv64/src/rv64v_settings.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) The mlkem-native project authors
+ * SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+ */
+#ifndef MLK_NATIVE_RISCV64_SRC_RV64V_SETTINGS_H
+#define MLK_NATIVE_RISCV64_SRC_RV64V_SETTINGS_H
+
+/*  === vector configuration */
+#ifndef MLK_RVV_VLEN
+#define MLK_RVV_VLEN 256
+#endif
+
+/*
+#define MLK_RVV_WIDENING_MUL
+*/
+
+/*  vl value for a 16-bit wide type */
+/* #define MLK_RVV_16M1_VL __riscv_vsetvlmax_e16m1() */
+#define MLK_RVV_E16M1_VL (MLK_RVV_VLEN / 16)
+
+/* check-magic: off */
+
+/*  Montgomery reduction constants */
+/*  n   = 256; q   = 3329; r   = 2^16 */
+/*  qi  = lift(Mod(-q, r)^-1) */
+#define MLKEM_QI 3327
+
+/*  r1  = lift(Mod(r, q)) */
+#define MLK_MONT_R1 2285
+
+/*  r2  = lift(Mod(r, q)^2) */
+#define MLK_MONT_R2 1353
+
+/*  in  = lift(Mod(n / 2, q)^-1) */
+/*  nr  = (in * r^2) % q */
+#define MLK_MONT_NR 1441
+
+/* check-magic: on */
```

Comment:
```suggestion
/* check-magic: 3327 == pow(-MLKEM_Q, -1, 2^16) */
#define MLKEM_QI 3327

/* check-magic: 2285 == unsigned_mod(2^16, MLKEM_Q) */
#define MLK_MONT_R1 2285

/* check-magic: 1353 == pow(MLK_MONT_R1, 2, MLKEM_Q) */
#define MLK_MONT_R2 1353

/* check-magic: 1441 == pow(2,32 - 7,MLKEM_Q) */
#define MLK_MONT_NR 1441

```

This auto-checks the magic number explanations in CI.